gulpfile.ts

```diff
@@ -1,10 +1,18 @@
 import { task, series, parallel } from 'gulp'
 import * as path from 'path'
+import * as tsPaths from 'tsconfig-paths'
+
+const { compilerOptions } = require('./build/tsconfig.common.json')
 
 // add node_modules/.bin to the path so we can invoke .bin CLIs in tasks
 process.env.PATH =
   process.env.PATH + path.delimiter + path.resolve(__dirname, 'node_modules', '.bin')
 
+tsPaths.register({
+  baseUrl: '.',
```

Comment:
shouldn't we use repo's path from config constant here as well? This will ensure consistency in the approach taken (there will be always single source of truth maintained)

Reply:
Did it 👍